HISTORY.md

```diff
@@ -11,6 +11,15 @@ The third number is for emergencies when we need to start branches for older rel
 
 Our backwards-compatibility policy can be found [here](https://github.com/python-attrs/cattrs/blob/main/.github/SECURITY.md).
 
+## 25.2.0 (unreleased)
+
+- Add a `use_alias` parameter to {class}`cattrs.Converter`. 
+  {py:func}`cattrs.gen.make_dict_unstructure_fn_from_attrs`, {py:func}`cattrs.gen.make_dict_unstructure_fn`,
```

Comment:
I don't think you need to use `{py:func}` here, just `{func}` should be enough I think.

Reply:
OK, dropped the `py` in all five places. Note, however, that I simply adapted from what was written in earlier parts of the changelog, where the `py` is present